lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -380,7 +384,7 @@ def inner(self, default_store, module_count, mongo_calls, sql_queries, *args, **
 
     @ddt.data(
         (ModuleStoreEnum.Type.mongo, 3, 4, 31),
-        (ModuleStoreEnum.Type.split, 3, 13, 31),
+        (ModuleStoreEnum.Type.split, 3, 12, 31),
```

Comment:
Something we changed ended up reducing the query count. I think @staubina looked into it but couldn't find the exact spot, since we never explicitly deleted any queries. I thought it might have something to do with caching, but I don't really know how everything interacts well enough yet to know if that's actually the case.